third_party/intel/include/Analysis/DPAS.h

```diff
@@ -0,0 +1,60 @@
+#ifndef TRITON_ANALYSIS_INTEL_DPAS_H
+#define TRITON_ANALYSIS_INTEL_DPAS_H
```

Comment:
```suggestion
#ifndef TRITON_INTEL_ANALYSIS_DPAS_H
#define TRITON_INTEL_ANALYSIS_DPAS_H
```

Shouldn't it be like this?

Reply:
done